vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +235,28 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(const std::string &packagerHost, const uint16_t packagerPort) {
+#if defined(HERMES_ENABLE_DEBUGGER)
+  std::string packageName("RNW"); 
+  if (winrt::Windows::ApplicationModel::Package::Current())
+    std::string packageName = winrt::to_string(winrt::Windows::ApplicationModel::Package::Current().DisplayName());
```

Comment:
Is this meant to reassign the above instead of declaring a shadowed variable?

Reply:
Ah .. Thanks for spotting this !